### DISCUSSION TITLE
Controls InteractionChange property closed

### DISCUSSION THREAD

**Discussion by omeryilmazinfo** [bcakmakoglu/vue-flow#472] [Q&A] [answered]
hello, I try to fixed panel object (as mount the project, anybody don't change the draft), and add controls in :on-interaction-change="false" , it didn't work, Could you help me please?

**Comment by bcakmakoglu** [comment, marked as answer]
Could you elaborate what you want to do specifically?
You want to disable all interactions?
Adding `:on-interaction-change="false"` doesn't do anything ^^
First you'd have to use it as `@interaction-change="handleInteractionChange"` as it's an event that's emitted and not a prop you can pass to the Controls.
Second, it listening to the event does not block interactions.

If you want to disable interactions you can use the `setInteractive` function of `useVueFlow`

```js
const { setInteractive } = useVueFlow()

setInteractive(false)
```
